python/scripts/worker-populate.py

```diff
@@ -41,13 +41,16 @@ def main(argv):
     run_daemon = True # execute at least one loop
     while run_daemon:
         populated_at_least_one = False
+        start = time.time()
         for name, rel in rels_cls.items():
             todo, done = rel().progress()
```

Comment:
progress can be a time consuming operator and its function is duplicated in populate.  I would just run populate without checking progress.